src/recipe.jl

```diff
@@ -0,0 +1,2 @@
+function plotcamerapath end
+function plotcamerapath! end
```

Comment:
Should we also move the stubs for `capture_view` and `set_view!` here? We might also want to add `plotcamerapath*` to the error hint in `__init__`, I find it's a frequent source of confusion to forget to load the "backend" and having the message helps anyone who is paying attention (which is not everyone, but...).

Reply:
Will do!  Yeah, I'll probably rename this file to `makie_interface.jl` then.

Reply:
I think of it more as "interface functions." These are the functions that any backend should provide implementations of.

Reply:
Perhaps the docs should have a page on implementing support for other backends?